test/preview/components/file_converters/test_markdown_to_document.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from haystack.preview.components.file_converters.markdown import MarkdownToTextDocument
+
+
+class TestMarkdownToTextDocument:
+    @pytest.mark.unit
```

Comment:
These and the following tests look like `integration' tests to me.
Please change the mark...

(I know there are confusing examples in the codebase)